buildSrc/src/main/java/Configuration.kt

```diff
@@ -0,0 +1,24 @@
+object Flavors {
+    const val PRODUCTION = "production"
+    const val STAGING = "staging"
+    const val DIMENSION_VERSION = "version"
+}
+
+object BuildTypes {
+    const val DEBUG = "debug"
+    const val RELEASE = "release"
+}
+
+object Modules {
+    const val SHARED = ":shared"
+}
+
+object Plugins {
+    const val ANDROID = "android"
+    const val ANDROID_APPLICATION = "com.android.application"
+    const val ANDROID_LIBRARY = "com.android.library"
+
+    const val COCOAPODS = "native.cocoapods"
```

Comment:
Should we use `IOS_COCOAPODS` as we are using `ANDROID_xxx` above?🙏

Reply:
Updated in https://github.com/nimblehq/kmm-templates/pull/25/commits/28f52e9f0fccca5bb530e2c905b90b030c46112c